[torch] fix comment typo

wilson100hong opened the pull request:
Summary: as title

Differential Revision: D49841787

zhxchen17 commented on torch/export/_trace.py:
```diff
@@ -506,7 +506,7 @@ def _export(
 
     Args:
-        m: the `nn.Module` or callable to trace.
+        f: the callable to trace.
```
Actually, this should only support nn.Module, not the other way around

zhxchen17 replied:
We can also change the type annotation of `f` to be torch.nn.Module here.

zhxchen17 reviewed (changes requested):
the doc needs some update

zhxchen17 reviewed (approved):
thanks!

wilson400hong committed:
[torch] fix comment typo (#118656)

Summary:

as title

Test Plan: n/a

Reviewed By: aay-gup, zhxchen17

Differential Revision: D49841787

zhxchen17 commented:
@pytorchbot merge

zhxchen17 commented:
@pytorchbot merge
